share/doc/homebrew/Rename-A-Formula.md

```diff
@@ -1,23 +1,14 @@
 # Renaming a Formula
 
-Sometimes software and formulae need to be renamed. To rename core formula
+Sometimes software and formulae need to be renamed. To rename a formula
 you need to:
 
 1. Rename the formula file and its class to a new formula. The new name must meet all the usual rules of formula naming. Fix any test failures that may occur due to the stricter requirements for new formulae than existing formulae (i.e. `brew audit --strict` must pass for that formula).
 
-2. Create a pull request to the main Homebrew repository deleting the old formula file, adding the new formula file and add it to `Library/Homebrew/formula_renames.rb` with a commit message like `newack: renamed from ack`
+2. Create a pull request to the corresponding tap deleting the old formula file, adding the new formula file and add it to `formula_renames.json` with a commit message like `newack: renamed from ack`. Use canonical name (e.g. `ack` instead of `user/repo/ack`).
```

Comment:
Is "canonical" the right term to use here? I think "unqualified" would make more sense. If there is a "canonical" name, it seems like that might be the qualified `user/repo/ack` name.